HW_visualization.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[210]:
+
+
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+# In[129]:
+
+
+#first task
+#functions for reading gff and bed6 files
+def read_gff(path_to_gff_file):
+    df_gff = pd.read_csv(path_to_gff_file,
+                 sep='\t', 
+                 header=0,
+                 names=['chromosome', 'source', 'type', 'start', 'end', 'score', 'strand', 'phase', 'attributes'])
+    return df_gff
+
+df_gff = read_gff('/home/sweetlana/BioInf2022/Python/homework/HW5_Visualisation/rrna_annotation.gff')
+
+
+# In[130]:
+
+
+def read_bed6(path_to_bed_file):
+    df_bed6 = pd.read_csv(path_to_bed_file, 
+                      sep='\t',
+                      names=['chromosome', 'start', 'end', 'name', 'score', 'strand'])
+    return df_bed6
+
+df_bed6 = read_bed6('/home/sweetlana/BioInf2022/Python/homework/HW5_Visualisation/alignment.bed')
+
+
+# In[131]:
+
+
+#rename column 'attributes' to shorter one
+df_gff['attributes'] = df_gff['attributes'].apply(lambda x: x.split('Name=')[1].split('_')[0])
+df_gff
+
+
+# In[417]:
+
+
+#make a table of attributes grouped by chromosome and create barplot
+two_col = df_gff[['chromosome','attributes']]
+table_gff = two_col.groupby('chromosome').value_counts().unstack()
+table_gff.plot.bar()
+
+
+# In[196]:
+
+
+#intersect two files and obtain information about contigs
+intersect = pd.merge(df_gff,df_bed6, on="chromosome", how="outer")
+intersect[(intersect['start_x'] >= intersect['start_y']) & (intersect['end_x'] <= intersect['end_y'])]
```

Comment:
Рабочий вариант, но мне больше нравится через `query`, так более лаконично
```suggestion
intersect.query("start_x >= start_y and end_x <= end_y")
```